queries.sql

```diff
@@ -25,24 +25,35 @@ BEGIN;
 
     SELECT * FROM animals;
 
-ROLLBACK;
+    ROLLBACK;
+
+COMMIT;
```

Comment:
- [ ] Kindly remove COMMIT which is after the ROLLBACK because when you conclude a transaction then you either use `ROLLBACK` **(To cancel the transaction)** or `COMMIT` **(To conclude the transaction to save it into real database)** but you can't use both together 😨